xpdan/db_utils.py

```diff
@@ -147,17 +147,25 @@ def query_dark(db, docs, schema=1):
 def query_background(db, docs, schema=1):
     if schema == 1:
         doc = docs[0]
-        return db(sample_name=doc['bkgd_sample_name'],
-                  is_dark={'$exists': False})
+        sample_name = doc.get('bkgd_sample_name')
```

Comment:
Be a bit more graceful about pulling the background from the document.

Reply:
👍